cpp/foxglove-websocket/include/foxglove/websocket/websocket_client.hpp

```diff
@@ -23,6 +23,12 @@ inline void to_json(nlohmann::json& j, const ClientAdvertisement& p) {
                      {"topic", p.topic},
                      {"encoding", p.encoding},
                      {"schemaName", p.schemaName}};
+  if (p.schema) {
+    j["schema"] = *p.schema;
+  }
+  if (p.schemaEncoding) {
+    j["schemaEncoding"] = *p.schemaEncoding;
+  }
```

Comment:
```suggestion
  if (p.schema) {
    j["schema"] = p.schema.value();
  }
  if (p.schemaEncoding) {
    j["schemaEncoding"] = p.schemaEncoding.value();
  }
```

To be consistent with the rest of the codebase